patches/components-permissions-permission_uma_util.cc.patch

```diff
@@ -10,3 +10,11 @@ index 800f537d9e4480a15d73e854709f0d503c3a3c6a..87f41225c5cd8de80546ff0a1922a767
    }
  }
  
+@@ -762,6 +763,7 @@ void PermissionUmaUtil::RecordPermissionAction(
+       base::UmaHistogramEnumeration("Permissions.Action.FileHandling", action,
+                                     PermissionAction::NUM);
+       break;
++    BRAVE_RECORD_PERMISSION_ACTION
```

Comment:
is this for a runtime error or build time error?

Reply:
this one is runtime error when we falls into default and hit NOTREACHED, I'll look this one again if we could avoid this by override or sth

Reply:
I think we can replace ```GetUkmSourceId``` call in this file with something no-op-like ```GetSettingsMap``` and then just do ```return```, because our UKM/histograms policy is quite strict and we're very unlikely to use these calls at all.
```
#define GetUkmSourceId             \
  GetSettingsMap(browser_context); \
  if (true)                        \
    return;                        \
  PermissionsClient::Get()->GetUkmSourceId
```

Reply:
if this is a runtime error we can just override `GetUkmSourceId` for the permissions client implementation

Reply:
Fixed in https://github.com/brave/brave-core/pull/9443/commits/b127eefe84bac3157ee00dbdfef764ebf6a691a6